pattern-distributed/README.md

```diff
@@ -11,12 +11,6 @@ Core Kubernetes resources for a [fully distributed deployment of WSO2 Stream Pro
 
 ## Prerequisites
 
-* In order to use WSO2 Kubernetes resources, you need an active WSO2 subscription. If you do not possess an active
-WSO2 subscription already, you can sign up for a WSO2 Free Trial Subscription from [here](https://wso2.com/free-trial-subscription).<br><br>
-
-* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (compatible with v1.10)
```

Comment:
@ThilinaManamgoda why is this prerequisite removed? We do need these clients to perform the installation.

Reply:
Yes, clients should be there. I'll add them back.